locales/en.default.json

```diff
@@ -203,7 +203,11 @@
         "total": "Total",
         "quantity": "Quantity"
       },
-      "update": "Update"
+      "update": "Update",
+      "login": {
+        "title": "Have an account?",
+        "paragraph_html": "<a href=\"{{ link }}\" class=\"link underlined-link\">Log in</a> to check out faster."
+      }
```

Comment:
Weird. The translation platform says "good to go". But this is new.

Reply:
Thanks for the help for troubleshooting. I'm requesting them now 👍